stdlib/deplinker/__init__.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+"""Provide different dependency linkers that can be used interchangeably as part of a template.
+
+All dependency linkers are python functions that try to assign requirements to the generated packages automatically.
+They analyse the content of the package and do their best to figure out how those new packages are related to the other existing ones.
+
+Even though they might not always find all requirements, they provide a solide base to avoid mistakes and obvious, redundant requirements.
```

Comment:
```suggestion
Even though they might not always find all requirements, they provide a solid base to avoid mistakes and obvious, redundant requirements.
```